cmd/fakedoc/main.go

```diff
@@ -24,6 +23,8 @@ import (
 	"strings"
 	"text/template"
 
+	"github.com/spf13/pflag"
```

Comment:
The other tools from gocsaf are using `github.com/jessevdk/go-flags`. We should consider it for symmetry.